src/rgw/rgw_rest_s3.cc

```diff
@@ -429,10 +429,20 @@ void RGWGetUsage_ObjStore_S3::send_response()
        formatter->dump_int("SuccessfulOps", total_usage.successful_ops);
        formatter->close_section(); // total 
        formatter->close_section(); // user
-     } 
-     formatter->dump_int("TotalBytes", header.stats.total_bytes);
-     formatter->dump_int("TotalBytesRounded", header.stats.total_bytes_rounded);
-     formatter->dump_int("TotalEntries", header.stats.total_entries);
+     }
+
+     if (s->cct->_conf->rgw_getusage_dump_description) {
+       formatter->open_object_section("Stats"); 	
```

Comment:
@weiqiaomiao close if here, and reopen it for the close_section(), so that we don't duplicate the stats dump